### PR TITLE
Paypal test

### PR DESCRIPTION
**Pull request by shawnhankim**
#### Any background context you want to provide?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
```
Comment (thomasf) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: remove these from all markdown files

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 .idea
-.vscode
\ No newline at end of file
+.vscode
+secret.go
```
Comment (thomasf) [inline .gitignore:3]: what is secret.go?

File: go.mod [hunk 3]
```diff
@@ -1,5 +1,8 @@
-module github.com/plutov/paypal/v4
+module github.com/shawnhankim/paypal
```
Comment (thomasf) [inline go.mod:1]: wrong module path

File: go.sum [hunk 4]
```diff
@@ -1,5 +1,7 @@
 github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/plutov/paypal/v4 v4.1.0 h1:tyf+UX5s5LRvHcX8N5ED0yekAAfkt5rBYSYYYsm8yjg=
```
Comment (thomasf) [inline go.sum:3]: the project should not depend on itself

File: mycodes/01_get_access_token_test.go [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package main
```
Comment (thomasf) [inline mycodes/01_get_access_token_test.go:1]: don't create a new directory for tests.
don't prefix file names with numbers.

File: mycodes/util/utils.go [hunk 6]
```diff
@@ -0,0 +1,68 @@
+package util
```
Comment (thomasf) [inline mycodes/util/utils.go:1]: you don't need a separate package for this, in the main package, all _test.go files can share code between each other.

File: mycurls/01_paypal_curl.txt [hunk 7]
```diff
@@ -0,0 +1,43 @@
+curl -v -X POST https://api-m.sandbox.paypal.com/v2/checkout/orders \
```
Comment (thomasf) [inline mycurls/01_paypal_curl.txt:1]: It is unclear what purpose these files serve, they look like local development notes that should not be committed into the repository.

**Review by thomasf — COMMENTED** [review]
what are you trying to achieve? There is a file for integration tests already, if you want to add more tests you should probably add them there.

**Review by plutov — COMMENTED** [review]
Closing the PR, since the description says nothing, and it's not clear what this code is doing.

**Comment by shawnhankim** [comment]
Thanks for closing this PR. I apologize for this inconvenience. It was accidentally raised while testing PayPal API.

**Pull request closed without merging**